vllm/engine/llm_engine.py

```diff
@@ -1616,7 +1616,24 @@ def _get_stats(self,
         num_generation_tokens_requests: List[int] = []
         n_requests: List[int] = []
         finished_reason_requests: List[str] = []
-
+        
+        # Lora requests
+        running_adapters = dict(collectionsCounter([
+            running_request.lora_request.lora_name
+            for scheduler in self.scheduler
+            for running_request in scheduler.running
+            if running_request.lora_request
+            ]))
+        waiting_adapters = dict(collectionsCounter([
+            waiting_request.lora_request.lora_name
+            for scheduler in self.scheduler
+            for waiting_request in scheduler.waiting
+            if waiting_request.lora_request
+            ]))
+        max_lora_stat = "0"
+        if self.lora_config:
+            max_lora_stat = str(self.lora_config.max_loras)
```

Comment:
This seems always fixed? In this case can we don't dump this value?

Reply:
across multiple deployments its hard to get this value, helps determine how many loras can be fitted on the server. You are right, its definitely static right now, initialised at runtime and thats it. I considered moving it to separate info metric like the cache config info. But I think in future there maybe value in enabling dynamic adjustment of max lora, like base_model which is static right now.